openapi/zuora.collection/Module.md

```diff
@@ -0,0 +1,21 @@
+## Overview
+    This is a generated connector for [Zuora Collections API V1](https://www.zuora.com/developer/collect-api/#) OpenAPI specification.
+  
+    This Zuora Collections API provides detailed descriptions about functions, requests, and responses of
+    the APIs for the following two component apps. 
+
+    - Advanced Payment Manager
+
+    - Collections Window
+
+    - Configurable Payment Retry
+
+    - Statement Generator
```

Comment:
,We mention as `following two component apps`, but there are 4 components given below